### ISSUE TITLE
Roster view in team stats doesn't show the ORtgs (other fields?) correctly

### ISSUE THREAD

**Issue by Alex-At-Home** [Alex-At-Home/cbb-on-off-analyzer#220]
Eg http://localhost:3000/OnOffAnalyzer?autoOffQuery=true&baseQuery=&gender=Men&maxRank=400&minRank=0&offQuery=NOT%20%28Emilien%20%29&onQuery=Emilien%20&showRoster=true&team=Maryland&year=2022%2F23&

Check out Emilien, Reese on/off etc

**Comment by Alex-At-Home** [comment]
OK it's a bit tricky ... I think the issue is `teamStats.baseline` which should be the on/off number instead

```ts
            const playerInfo = LineupTableUtils.buildBaselinePlayerInfo(
              playerStatsBy0AB, globalRosterStatsByCode, teamStats.baseline, avgEfficiency, adjustForLuck, 
              luckConfig.base, manualOverridesAsMap || {}
            );
```

**Comment by Alex-At-Home** [comment]
Fixed by 6674bc3aedf3ae37df447f5611b4b16f32b6c7b7
